### PR TITLE
txindex: only restart bitcoind when running

### LINKED ISSUE #1830: [edge case] restarting bitcoind during the recovery process freezes
This is an edge case only if the recovery process starts with a new blockchain database without txindex, but BTC-RPC-Explorer enabled.
On activating `txindex` and restarting bitcoind the process freezes same as with the `blockfilterindex` in :https://github.com/rootzoll/raspiblitz/issues/1699
```
# *** INSTALL BTC-RPC-EXPLORER ***
nodeJS is already installed
Installed nodeJS v14.15.0
switching txindex=1 and restarting bitcoind
```
Ran
`sudo systemctl stop bitcoind`
manually for the process to continue.
```
Job for bitcoind.service canceled.
```

### PR DESCRIPTION
**Pull request by openoms**
fixes: https://github.com/rootzoll/raspiblitz/issues/1830

An edge case only if the recovery process starts with a new blockchain database without txindex, but BTC-RPC-Explorer enabled.

### CODE REVIEW

File: home.admin/config.scripts/network.txindex.sh [hunk 1]
```diff
@@ -64,17 +64,22 @@ fi
...
-
   # check txindex (parsed and sourced from bitcoin network config above)
   if [ ${txindex} == 0 ]; then
     sudo sed -i "s/^txindex=.*/txindex=1/g" /mnt/hdd/${network}/${network}.conf
-    echo "switching txindex=1 and restarting ${network}d"
-    sudo systemctl restart ${network}d
-    echo "The indexing takes ~7h on an RPi4 with SSD"
-    echo "monitor with: sudo tail -n 20 -f /mnt/hdd/${network}${pathAdd}/debug.log"
+    echo "# switching txindex=1"
+    isBitcoinRunning=$(sudo systemctl is-active ${network}d | grep -c "^active")
```
Comment (ghost) [inline home.admin/config.scripts/network.txindex.sh]: Pretty sure you can check the status without sudo

Comment (openoms) [reply home.admin/config.scripts/network.txindex.sh]: that's right, thanks!

**Comment by rootzoll** [comment]
@openoms can you rebase on `dev` branch?

**Comment by openoms** [comment]
ok, rebased to dev

**Pull request merged**